app/views/admins/_stats.haml

```diff
@@ -0,0 +1,50 @@
+- content_for :head do
+  = stylesheet_link_tag :admin
+
+%h1= t(".usage_statistic")
+
+.row
+  = form_tag("/admins/stats", method: "get", class: "col-sm-12 form-inline segment-selection") do
+    .col-sm-2.col-sm-push-8
+      %select.form-control{name: "range"}
+        - ["daily", "week", "2weeks", "month"].each do |range|
+          %option{value: range, selected: ("selected" if params[:range] == range)}
+            = t(".#{range}")
+    .col-sm-2.col-sm-push-8
+      = submit_tag t(".go"), class: "btn btn-primary btn-block"
+
+  .col-sm-8.col-sm-pull-4
+    %h4
+      != t("admins.stats.display_results", segment: content_tag(:strong, @segment))
```

Comment:
Include the partial like this:

```haml
= render partial: "admins/stats", locals: {segment: @segment}
```

and then use `segment` here instead of `@segment`